test/lib/adapter.test.ts

```diff
@@ -163,103 +233,90 @@ describe('adapter', () => {
       expect(app.adapters[0].instance).toBeTruthy()
-      app.adapters[0].instance.emit('message', msg, RAW_CONN)  
+      app.adapters[0].instance.emit('message', msg, RAW_CONN)
     })
   })
-  
+
   describe('on("server:ready")', () => {
     it('notifies the Glee app', async () => {
-      const app = new Glee()
-      const adapter = new GleeAdapter(app, TEST_SERVER_NAME, TEST_SERVER, TEST_ASYNCAPI_DOCUMENT)
-      app.on('adapter:server:ready', (ev) => {
-        expect(ev).toStrictEqual({
-          fake: 'object',
-          serverName: TEST_SERVER_NAME,
-          server: TEST_SERVER,
-        })
-      })
+      const adapter = initializeAdapter()
       adapter.emit('server:ready', { fake: 'object' })
     })
   })
-  
+
   describe('on("server:connection:open")', () => {
     it('notifies the Glee app', async () => {
-      const app = new Glee()
-      const adapter = new GleeAdapter(app, TEST_SERVER_NAME, TEST_SERVER, TEST_ASYNCAPI_DOCUMENT)
-      app.on('adapter:server:connection:open', (ev) => {
-        expect(ev.serverName).toStrictEqual(TEST_SERVER_NAME)
-        expect(ev.server).toStrictEqual(TEST_SERVER)
-        expect(ev.connection).toBeInstanceOf(GleeConnection)
-        expect(ev.connection.AsyncAPIServer).toStrictEqual(TEST_SERVER)
-        expect(ev.connection.rawConnection).toStrictEqual(RAW_CONN)
-        expect(ev.connection.channels).toStrictEqual(['fake/channel'])
-        expect(ev.connection.serverName).toStrictEqual(TEST_SERVER_NAME)
-        expect(ev.connection.parsedAsyncAPI).toStrictEqual(TEST_ASYNCAPI_DOCUMENT)
+      const adapter = initializeAdapter()
+      adapter.emit('server:connection:open', {
+        channels: ['fake/channel'],
+        connection: RAW_CONN,
       })
-      adapter.emit('server:connection:open', { channels: ['fake/channel'], connection: RAW_CONN })
     })
   })
-  
+
   describe('on("close")', () => {
     it('notifies the Glee app', async () => {
-      const app = new Glee()
-      const adapter = new GleeAdapter(app, TEST_SERVER_NAME, TEST_SERVER, TEST_ASYNCAPI_DOCUMENT)
-      app.on('adapter:close', (ev) => {
-        expect(ev.serverName).toStrictEqual(TEST_SERVER_NAME)
-        expect(ev.server).toStrictEqual(TEST_SERVER)
-        expect(ev.connection).toBeInstanceOf(GleeConnection)
-        expect(ev.connection.AsyncAPIServer).toStrictEqual(TEST_SERVER)
-        expect(ev.connection.rawConnection).toStrictEqual(RAW_CONN)
-        expect(ev.connection.channels).toStrictEqual(['fake/channel'])
-        expect(ev.connection.serverName).toStrictEqual(TEST_SERVER_NAME)
-        expect(ev.connection.parsedAsyncAPI).toStrictEqual(TEST_ASYNCAPI_DOCUMENT)
+      const adapter = initializeAdapter()
+      adapter.emit('close', {
+        channels: ['fake/channel'],
+        connection: RAW_CONN,
       })
-      adapter.emit('close', { channels: ['fake/channel'], connection: RAW_CONN })
     })
   })
-  
+
   describe('on("reconnect")', () => {
```

Comment:
Why are we removing this test?

Reply:
it's not removed, I refactored it. It was making use of the same logic so I moved that logic to a separate function because it was causing sonar to fail due to code duplication.